paddle/fluid/operators/conv_op.cc

```diff
@@ -122,6 +128,11 @@ void Conv2DOpMaker::Make() {
            "H is the height of the filter, and W is the width of the filter. "
            "If the groups attribute is greater than 1, C equals the number of "
            "input image channels divided by the groups.");
+  AddInput("Bias",
+           "(Tensor) Bias to be added to each output of filter application."
+           "The format of output tensor is X (one-dimensional) of size equal"
+           "to the number of output channels.")
```

Comment:
Could you update the annotation of "Bias", make users know "Bias" in only used in MKLDNN?